metricbeat/module/http/json/json.go

```diff
@@ -150,6 +135,48 @@ func (m *MetricSet) Fetch() (common.MapStr, error) {
 	// Set dynamic namespace
 	event["_namespace"] = m.namespace
 
+	return event
+}
+
+// Fetch methods implements the data gathering and data conversion to the right format
+// It returns the event which is then forward to the output. In case of an error, a
+// descriptive error must be returned.
+func (m *MetricSet) Fetch() ([]common.MapStr, error) {
+	response, err := m.http.FetchResponse()
+	if err != nil {
+		return nil, err
+	}
+	defer response.Body.Close()
+
+	var jsonBody interface{}
+	var event []common.MapStr
```

Comment:
`event` should be named `events` as this now an array of events

Reply:
@christiangalsterer glad you stumbled over the PR.